.gitignore

```diff
@@ -8,3 +8,6 @@ media
 property_images/
 misc/
 **/.DS_Store
+node_modules/
+frontend/.env.example
```

Comment:
The frontend .gitignore is the thing that ignores node_modules. we shouldnt specify it here

https://github.com/daxaxelrod/open_insure/blob/master/frontend/.gitignore

we def want to keep the .env.examples in the git history, otherwise new devs wont have a reference to setup their envs